spec/lib/dispatch-rider/subscriber_spec.rb

```diff
@@ -120,16 +131,17 @@ def process(options)
 
       context "when process is interupted twice" do
         before do
-          subject.demultiplexer.stub(:stop) # do nothing just ignore the interuption
-          subject.stub(:exit){ throw :got_forcefully_stopped }
+          allow(subject.demultiplexer).to receive(:stop) # do nothing just ignore the interuption
+          allow(subject).to receive(:exit) { throw :got_forcefully_stopped }
 
           stub_const("TwiceInterupter", Class.new(DispatchRider::Handlers::Base) {
-            def process(options)
-              2.times { Process.kill("INT", 0) }
-            end
-          })
+                                        def process(options)
+                                          2.times { Process.kill("INT", 0) }
+                                        end
+                                      })
```

Comment:
`end` at 141, 38 is not aligned with `Class.new(DispatchRider::Handlers::Base) {` at 137, 40 or `stub_const("TwiceInterupter", Class.new(DispatchRider::Handlers::Base) {` at 137, 10